sdk/synapse/Azure.Analytics.Synapse.Artifacts/CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Release History
 
-## 1.0.0-preview.1 (Unreleased)
+## 1.0.0-preview.2 (Unreleased)
+
+- Added support for long running operation.
+
+- Added generated model classes.
+
+## 1.0.0-preview.1
```

Comment:
Nit - you should add the date of your last preview like `## 1.0.0-preview.1 (2020-06-31)` or whenever the actual date was.  That's the format we're using for all these.

Reply:
Fixed.